handlers_test.go

```diff
@@ -37,6 +37,96 @@ import (
 	margo "go.mozilla.org/mar"
 )
 
+type HandlerTestCase struct {
```

Comment:
I'm having difficulty seeing what's changed functionally in this test code. Could you point me to the actual differences from the original that isn't just refactoring?

Reply:
Whoops, sorry, I thought I deleted this one! I figured out what was going on!